pyproject.toml

```diff
@@ -7,17 +7,17 @@ readme = "README.adoc"
 license = "Apache-2.0"
 requires-python = ">=3.12,<3.13"
 dependencies = [
-    "ttyg==1.5.0",
+    "ttyg==1.7.0",
     "ttyg-evaluation==2.2.0",
     "jupyter==1.1.1",
-    "langchain-openai==0.3.27",
-    "langgraph-checkpoint-redis==0.0.8",
+    "langchain-openai==0.3.32",
+    "langgraph-checkpoint-redis==0.1.1",
     "jsonlines==4.0.0",
-    "cognite-sdk==7.76.0",
+    "cognite-sdk==7.83.0",
```

Comment:
Are you sure that this version of the SDK is compatible with the version of the Cognite that we'll be used?

Reply:
Yes, upgraded to `cognite-sdk` `7.83.1` and tested with the current running Cognite. Works as expected